src/Services/ProductsDisplayService.php

```diff
@@ -0,0 +1,17 @@
+<?php
+class ProductsDisplayService
+{
+    public static function displayProducts(ProductEntity $productEntity): string
```

Comment:
```suggestion
    public static function displayProduct(ProductEntity $productEntity): string
```

This actually displays *a* Product